packages/shorebird_cli/pubspec.yaml

```diff
@@ -33,7 +33,7 @@ dependencies:
   pem: ^2.0.5
   platform: ^3.1.6
   pointycastle: ^3.9.1
-  propertylistserialization: ^1.3.0
+  propertylistserialization: ^1.4.1
```

Comment:
looks like it's using a published version and not a fork (does the PR title need to be updated)?

Reply:
Updated the PR